src/Kdyby/Translation/Console/CreateTableCommand.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Translation\Console;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Doctrine\DBAL\Types\Type;
+use Kdyby;
+use Nette;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+
+
+/**
+ * @author Filip Procházka <filip@prochazka.su>
+ */
+class CreateTableCommand extends Command
+{
+
+	/** @var Nette\DI\Container */
+	private $serviceLocator;
+
+	/** @var Connection */
+	private $connection;
+
+	/** @var AbstractSchemaManager */
+	private $schemaManager;
+
+	/** @var string */
+	public $table;
+
+	/** @var string */
+	public $key;
+
+	/** @var string */
+	public $locale;
+
+	/** @var string */
+	public $message;
+
+	/** @var string */
+	public $updatedAt;
+
+	protected function configure()
+	{
+		$this->setName('kdyby:translation-create-table')
+			->setDescription('Builds query for creating of database table.')
+			->setDefinition(array(
+				new InputOption(
+					'dump-sql', null, InputOption::VALUE_NONE,
+					'Dumps the generated SQL statement to the screen (does not execute it).'
+				),
+				new InputOption(
+					'force', null, InputOption::VALUE_NONE,
+					'Causes the generated SQL statement to be physically executed against your database.'
+				)
+			));
+	}
+
+
+	protected function initialize(InputInterface $input, OutputInterface $output)
+	{
+		$this->serviceLocator = $this->getHelper('container')->getContainer();
+		$this->connection = $this->serviceLocator->getByType('Doctrine\DBAL\Connection');
+		$this->schemaManager = $this->serviceLocator->getByType('Doctrine\DBAL\Schema\AbstractSchemaManager');
+	}
+
+
+	protected function execute(InputInterface $input, OutputInterface $output)
+	{
+		if (!$input->getOption('dump-sql') && !$input->getOption('force')) {
+			$output->writeln('<error>You must run the command either with --dump-sql or --force.</error>');
+			return 0;
+		}
+
+		if (!$this->schemaManager->tablesExist($this->table)) {
```

Comment:
To jsem psal [taky](https://github.com/Kdyby/Translation/pull/86#discussion_r38814768). :)